ResearchKit/ActiveTasks/ORKAudioContentView.m

```diff
@@ -47,7 +47,7 @@ @interface ORKAudioGraphView : UIView
 @property (nonatomic, strong) UIColor *keyColor;
 @property (nonatomic, strong) UIColor *alertColor;
 
-@property (nonatomic, copy) NSArray *values;
+@property (nonatomic, strong) NSArray *values;
```

Comment:
I believe this was appropriately copy. Calling code passing a mutable array of NSValue could subsequently append to that array.